src/main/java/top/jsoft/homework/jdbc/db/DataBaseConnection.java

```diff
@@ -0,0 +1,77 @@
+package top.jsoft.homework.jdbc.db;
+
+import org.mariadb.jdbc.MariaDbPoolDataSource;
+import top.jsoft.commons.util.PrintManager;
+import top.jsoft.homework.jdbc.Config;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+/**
+ * Created by psygrammator
+ * group jsoft.top
+ */
+public class DataBaseConnection {
```

Comment:
Better to implement AutoClosable Interface

Reply:
Done, thanks